CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ For notes on migrating to 2.x / 0.200.x see [the upgrade guide](doc/upgrade-to-2
 
 ## Unreleased
 
+* feat(sampler-composite): Added experimental implementations of draft composite sampling spec [#5839](https://github.com/open-telemetry/opentelemetry-js/pull/5839) @anuraaga
+
```

Comment:
Please move this entry to "experimental/CHANGELOG.md" -- the changelog file for the packages under experimental/...